### DISCUSSION TITLE
[Idea] 虚拟开源社区、虚拟开源基金会

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#316] [1️⃣ 新鲜事]
随着 AIGC 技术的发展，Agent 开始走向越来越多的场景，包括软件开发、数据分析、内容生成等。这些智能体形成了一个多智能体组织结构，其使命是“**通过编程改变数字世界**”。ChatDev 内的智能体通过参加专业的功能研讨会来 协作，包括设计、编码、测试和文档编写等任务。

例如，[ChatDev ](https://github.com/OpenBMB/ChatDev)是一家虚拟软件公司（开源项目），通过各种不同角色的智能体运营，包括 CEO、技术官、程序员、测试员等。

往开源领域延伸一下，虚拟开源社区、虚拟开源基金会，也许能够发挥我们在开源领域知识和开源行为数据方面的优势~

**Comment by bifenglin** [comment]
虚拟软件公司的产出目标是软件，开源社区的产出也是软件，本质上我认为是相同的。区别可能就是群智范围更广？拥有不同领域技能的人更多一些？

**Comment by bifenglin** [comment]
但是我认为在开源领域，GPT的模型众包是一个很不错的概念，如何有效的评测分配，如何使用Agent管理众包内容感觉是一个点，但是目前没有相关研究参考，看看有没有感兴趣的同学。

**Comment by Wuhanqin** [comment]
我感觉这个可以跟开源数据分析结合起来玩
